### PR TITLE
DISTX-591. Add entitlement for GCP DataHub AutoScaling.

### PR DESCRIPTION
**Pull request by sidseth**
- Entitle String: DATAHUB_GCP_AUTOSCALING

See detailed description in the commit message.

### CODE REVIEW

**Review by doktoric — APPROVED** [review]

**Review by smaniraju — APPROVED** [review]
LGTM

**Comment by sidseth** [comment]
retest

**Comment by sidseth** [comment]
retest this please

**Review by cegganesh84 — APPROVED** [review]
LGTM

File: mock-thunderhead/src/test/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementServiceTest.java [hunk 1]
```diff
@@ -174,7 +174,7 @@ void getAccountTestIncludesFixedEntitlements() {
         Account account = res.getAccount();
         List<String> entitlements = account.getEntitlementsList().stream().map(Entitlement::getEntitlementName).collect(Collectors.toList());
         assertThat(entitlements).contains("CDP_AZURE", "CDP_GCP", "CDP_AUTOMATIC_USERSYNC_POLLER", "CLOUDERA_INTERNAL_ACCOUNT", "DATAHUB_AZURE_AUTOSCALING",
-                "DATAHUB_AWS_AUTOSCALING", "LOCAL_DEV", "DATAHUB_FLOW_SCALING", "DATAHUB_STREAMING_SCALING", "CDP_CM_ADMIN_CREDENTIALS");
+                "DATAHUB_AWS_AUTOSCALING", "DATAHUB_GCP_AUTOSCALING", "LOCAL_DEV", "DATAHUB_FLOW_SCALING", "DATAHUB_STREAMING_SCALING", "CDP_CM_ADMIN_CREDENTIALS");
```
Comment (cegganesh84) [inline mock-thunderhead/src/test/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementServiceTest.java:177]: Checkstyle is complaining that this line is too long.

Comment (sidseth) [reply mock-thunderhead/src/test/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementServiceTest.java:177]: Thank you. The previous test result had expired.
Now the specific line - do you figure that out from the Jenkins run, or a local run? (Don't see any output artifacts on jenkins)

Comment (cegganesh84) [reply mock-thunderhead/src/test/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementServiceTest.java]: It was the Jenkins run, had to scroll a little bit above to see the actual error printed in stdout itself.

**Pull request merged**